app/controllers/api/v1/users_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::UsersController < ApplicationController
+  load_and_authorize_resource
```

Comment:
We should also ensure the reservations_controller and fitness_activities_controller actions are authorized! by also including this `load_and_authorize_resource` before their actions